packages/webview_flutter/ios/Tests/FLTWebViewTests.m

```diff
@@ -64,4 +66,22 @@ - (void)webViewScrollIndicatorAticautomaticallyAdjustsScrollIndicatorInsetsShoul
   }
 }
 
+- (void)testContentInsetsSumAlwaysZeroAfterSetFrame {
+  FLTWKWebView *webView = [[FLTWKWebView alloc] initWithFrame:CGRectMake(0, 0, 300, 400)];
+  webView.scrollView.contentInset = UIEdgeInsetsMake(0, 0, 300, 0);
+  XCTAssertFalse(UIEdgeInsetsEqualToEdgeInsets(webView.scrollView.contentInset, UIEdgeInsetsZero));
+  webView.frame = CGRectMake(0, 0, 300, 200);
```

Comment:
Lets also assert that the frame was updated, as we are overriding the setter.

Reply:
done